libraries/utilities/generate_clusters_from_pool.py

```diff
@@ -420,8 +420,10 @@ def generate_clusters_from_pool(pool_file):
         ClusterDef("3sg_2ac_3cbs_3lgs", num_sgs=3, num_acs=2, num_cbs=3, num_lgs=3, num_lbs=0),
         ClusterDef("4sg_1ac_3cbs_4lgs", num_sgs=4, num_acs=1, num_cbs=3, num_lgs=4, num_lbs=0),
         ClusterDef("4sg_2ac_3cbs_4lgs", num_sgs=4, num_acs=2, num_cbs=3, num_lgs=4, num_lbs=0),
+        ClusterDef("4sg_8ac_3cbs_4lgs", num_sgs=4, num_acs=8, num_cbs=3, num_lgs=4, num_lbs=0),
         ClusterDef("8sg_1ac_3cbs_8lgs", num_sgs=8, num_acs=1, num_cbs=3, num_lgs=8, num_lbs=0),
         ClusterDef("8sg_2ac_3cbs_8lgs", num_sgs=8, num_acs=2, num_cbs=3, num_lgs=8, num_lbs=0),
+        ClusterDef("16sg_8ac_3cbs_16lgs", num_sgs=16, num_acs=8, num_cbs=3, num_lgs=16, num_lbs=0),
```

Comment:
This might break the test_generate_clusters_from_pool.py. It will need n update.